src/mux.c

```diff
@@ -137,60 +158,77 @@ static bool stick_deviated(mux_channel_t channel)
 
 static bool do_roll_pitch_yaw_muxing(mux_channel_t channel)
 {
-  bool rc_override;
+  bool rc_override = (rc_switch_mapped(RC_SWITCH_ATT_OVERRIDE) && rc_switch(RC_SWITCH_ATT_OVERRIDE))
+                      || stick_deviated(channel) || muxes[channel].rc->active;
```

Comment:
Won't this always go into rc override if not in failsafe mode since rc->active is always true if not in failsafe mode and you've got an OR here? I was really trying to move way from these active fields since in my opinion they make the logic more confusing; the only reason I didn't delete it entirely was because I needed some way of knowing which values were valid in the offboard control message.